# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bluez-hcidump/0002-Add-missing-const-to-utility-functions.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 54e0c6f46b2a13a66083a98831d515395fb43757 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Dieb=20Martins?= <andre.dieb@signove.com>
Date: Mon, 31 Jan 2011 16:09:32 -0300
Subject: [PATCH 2/6] Add missing const to utility functions

---
 parser/hci.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/parser/hci.c b/parser/hci.c
index 8299403..fbbebde 100644
--- a/parser/hci.c
+++ b/parser/hci.c
@@ -704,7 +704,7 @@ static char *authentication2str(uint8_t authentication)
 	}
 }
 
-static char *eventmask2str(uint8_t mask[8])
+static char *eventmask2str(const uint8_t mask[8])
 {
 	int i;
 
@@ -733,7 +733,7 @@ static char *eventmask2str(uint8_t mask[8])
 	}
 }
 
-static char *lefeatures2str(uint8_t features[8])
+static char *lefeatures2str(const uint8_t features[8])
 {
 	if (features[0] & 0x01)
 		return "Link Layer supports LE Encryption";
-- 
1.7.2.3

